brainkeeper/src/app/components/session_screen/big-person-card/big-person-card.component.scss

```diff
@@ -0,0 +1,79 @@
+@media  (min-width: 900px) {
+  .big-card-right {
+    max-width: 700px;
+    width: 60%;
+    height: 80%;
+    position: fixed;
+    top: calc(5%);
+    left: calc(30%);
+    bottom: calc(5%);
+    display: inline-block;
+    max-height: calc(80%);
+    z-index: 100;
+    box-shadow: -1px 0px 33px 16px rgba(70,235,70,1);
+  }
+  .big-card-wrong {
+    max-width: 700px;
+    width: 60%;
+    height: 80%;
+    position: fixed;
+    top: calc(5%);
+    left: calc(30%);
+    bottom: calc(5%);
+    max-height: calc(80%);
```

Comment:
Calc is not needed when only using percentages without actually calculating something, e.g. calc(50% + 20px)

Reply:
changed with commit: 3c092315bd02c68198ed286b95baee75aafa7339